### PR TITLE
Automatically build page with Travis-CI

### LINKED ISSUE #3: Let Travis-CI automatically build the page
...with every push on the source branch.

### PR DESCRIPTION
**Pull request by ubruhin**
Fixes #3

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - echo "librepcb.org" > public/CNAME # required for GitHub Pages
+
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN
+  local_dir: public
+  target_branch: master
+  on:
+    branch: source
```
Comment (rnestler) [inline .travis.yml]: How does this deploy stuff work? You just say which directory (public) you want to deploy to which target_branch?

**Comment by rnestler** [comment]
Looks nice :+1:

File: .travis.yml (hunk 1 above)
Comment (dbrgn) [inline .travis.yml]: Oh, that's nice :)

Comment (rnestler) [inline .travis.yml]: I should use this for my blog :)

Comment (ubruhin) [inline .travis.yml]: Yep, it's pretty cool :) You just need to create an access token for the GitHub repository to give Travis push access. I have created the account @LibrePCB-Builder for that purpose...

**Pull request merged**